source/manual/howto-restore-an-individual-postgres-database.html

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#re-govuk"
+title: Restore an Individual Postgres Database in AWS RDS
+section: Backups
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-12-10
+review_in: 3 months
+---
+
+> If you need to restore an entire RDS instance please see these [instructions](https://docs.publishing.service.gov.uk/manual/howto-backup-and-restore-in-aws-rds.html) instead.
+
+### Restoring From S3 Backups
+
+We take nightly database backups and save them to the `govuk-production-database-backups` S3 bucket inside the `postgresql-backend` folder.
+
+To restore a backup from the most recent copy follow the steps below:
+1. SSH into the db_admin machine. If you are using GDS CLI you can run the following command to ssh into the correct machine:
+```
```

Comment:
Code blocks need a blank line above them to correctly render as code blocks.

Reply:
Code blocks now fixed